### PR TITLE
removed reconfigure-kubelet page

### PR DESCRIPTION
**Pull request by hamees-sayed**
Fix Issue #31585

### CODE REVIEW

**Comment by hamees-sayed** [comment]
Also, @sftim should I only remove English pages that link to `reconfigure-kubelet` or any page irrespective of their language?

**Comment by sftim** [comment]
For this PR, please only change English pages.

File: content/en/blog/_posts/2018-06-26-kubernetes-1-11-release-announcement.md [hunk 1]
```diff
@@ -24,10 +24,6 @@ In this release, [IPVS-based in-cluster service load balancing](https://github.c
 
 [CoreDNS](https://coredns.io) is now available as a [cluster DNS add-on option](https://github.com/kubernetes/features/issues/427), and is the default when using kubeadm. CoreDNS is a flexible, extensible authoritative DNS server and directly integrates with the Kubernetes API. CoreDNS has fewer moving parts than the previous DNS server, since it’s a single executable and a single process, and supports flexible use cases by creating custom DNS entries. It’s also written in Go making it memory-safe. You can learn more about CoreDNS [here](https://youtu.be/dz9S7R8r5gw).
 
-## Dynamic Kubelet Configuration Moves to Beta
-
-This feature makes it possible for new Kubelet configurations to be rolled out in a live cluster.  Currently, Kubelets are configured via command-line flags, which makes it difficult to update Kubelet configurations in a running cluster. With this beta feature, [users can configure Kubelets in a live cluster](/docs/tasks/administer-cluster/reconfigure-kubelet/) via the API server.
-
```
Comment (sftim) [inline content/en/blog/_posts/2018-06-26-kubernetes-1-11-release-announcement.md:30]: We should leave this _section_ in (it's OK to remove the hyperlink, but leave the text as-is).

File: content/en/blog/_posts/2018-07-11-dynamic-kubelet-configuration.md [hunk 2]
```diff
@@ -43,8 +43,4 @@ Note that the default last-known-good configuration is the combination of Kubele
 
 See the following diagram for a high-level overview of a configuration update for a single Node:
 
-![kubelet-diagram](/images/blog/2018-07-11-dynamic-kubelet-configuration/kubelet-diagram.png)
-
-## How can I learn more?
-
-Please see the official tutorial at /docs/tasks/administer-cluster/reconfigure-kubelet/, which contains more in-depth details on user workflow, how a configuration becomes "last-known-good," how the Kubelet "checkpoints" config, and possible failure modes.
+![kubelet-diagram](/images/blog/2018-07-11-dynamic-kubelet-configuration/kubelet-diagram.png)
```
Comment (sftim) [inline content/en/blog/_posts/2018-07-11-dynamic-kubelet-configuration.md:46]: We should leave this text intact (it's OK to remove the hyperlink, but leave the text as-is).

File: content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md [hunk 3]
```diff
@@ -67,7 +67,7 @@ been deprecated. These removals have been superseded by newer, stable/generally
 
 ## API removals, deprecations, and other changes for Kubernetes 1.24
 
-* [Dynamic kubelet configuration](https://github.com/kubernetes/enhancements/issues/281): `DynamicKubeletConfig` is used to enable the dynamic configuration of the kubelet. The `DynamicKubeletConfig` flag was deprecated in Kubernetes 1.22. In v1.24, this feature gate will be removed from the kubelet. See [Reconfigure kubelet](/docs/tasks/administer-cluster/reconfigure-kubelet/). Refer to the ["Dynamic kubelet config is removed" KEP](https://github.com/kubernetes/enhancements/issues/281) for more information. 
+* [Dynamic kubelet configuration](https://github.com/kubernetes/enhancements/issues/281): `DynamicKubeletConfig` is used to enable the dynamic configuration of the kubelet. The `DynamicKubeletConfig` flag was deprecated in Kubernetes 1.22. In v1.24, this feature gate will be removed from the kubelet. Refer to the ["Dynamic kubelet config is removed" KEP](https://github.com/kubernetes/enhancements/issues/281) for more information.
```
Comment (sftim) [inline content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md:70]: We should leave this text intact (it's OK to remove the hyperlink, but leave the text as-is).

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 4]
```diff
@@ -862,7 +862,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `DynamicAuditing`: Used to enable dynamic auditing before v1.19.
 - `DynamicKubeletConfig`: Enable the dynamic configuration of kubelet. The
   feature is no longer supported outside of supported skew policy. The feature
-  gate was removed from kubelet in 1.24. See [Reconfigure kubelet](/docs/tasks/administer-cluster/reconfigure-kubelet/).
+  gate was removed from kubelet in 1.24.
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:865]:
```suggestion
- `DynamicKubeletConfig`: Enable dynamic configuration of the kubelet.
  (not available for kubelets from Kubernetes v1.24 or later).
```

**Review by sftim — COMMENTED** [review]
Actually, some more changes are needed. You have a couple of options:
- simplify the change just to the plain removal
- make further tweaks to ensure we don't change the _meaning_ of the existing blog articles.

**Comment by hamees-sayed** [comment]
Alright. In this commit, I retained the text in its original form and removed the hyperlinks.

File: content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md [hunk 5]
```diff
@@ -67,7 +67,7 @@ been deprecated. These removals have been superseded by newer, stable/generally
...
 * [Dynamic log sanitization](https://github.com/kubernetes/kubernetes/pull/107207): The experimental dynamic log sanitization feature is deprecated and will be removed in v1.24. This feature introduced a logging filter that could be applied to all Kubernetes system components logs to prevent various types of sensitive information from leaking via logs. Refer to [KEP-1753: Kubernetes system components logs sanitization](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1753-logs-sanitization#deprecation) for more information and an [alternative approach](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1753-logs-sanitization#alternatives=). 
 * [Removing Dockershim from kubelet](https://github.com/kubernetes/enhancements/issues/2221): the Container Runtime Interface (CRI) for Docker (i.e. Dockershim) is currently a built-in container runtime in the kubelet code base. It was deprecated in v1.20. As of v1.24, the kubelet will no longer have dockershim. Check out this blog on [what you need to do be ready for v1.24](/blog/2022/03/31/ready-for-dockershim-removal/).
```
Comment (Kart121) [inline content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md:72]: 'what you need to **do **be**** ready for v1.24 '   should be written as   'what you need **to be** ready for v1.24 ' in this line

File: content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md [hunk 6]
```diff
@@ -67,7 +67,7 @@ been deprecated. These removals have been superseded by newer, stable/generally
...
 * [Removing Dockershim from kubelet](https://github.com/kubernetes/enhancements/issues/2221): the Container Runtime Interface (CRI) for Docker (i.e. Dockershim) is currently a built-in container runtime in the kubelet code base. It was deprecated in v1.20. As of v1.24, the kubelet will no longer have dockershim. Check out this blog on [what you need to do be ready for v1.24](/blog/2022/03/31/ready-for-dockershim-removal/). 
 * [Storage capacity tracking for pod scheduling](https://github.com/kubernetes/enhancements/issues/1472): The CSIStorageCapacity API supports exposing currently available storage capacity via CSIStorageCapacity objects and enhances scheduling of pods that use CSI volumes with late binding. In v1.24, the CSIStorageCapacity API will be stable. The API graduating to stable initates the deprecation of the v1beta1 CSIStorageCapacity API. Refer to the [Storage Capacity Constraints for Pod Scheduling KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1472-storage-capacity-tracking) for more information.
```
Comment (Kart121) [inline content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md:73]: Spelling mistake noted : The API graduating to stable **initates** the deprecation of the v1beta1 CSIStorageCapacity API.

you would need to change it to **initiates**

**Review by Kart121 — CHANGES_REQUESTED** [review]

File: content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md (hunk 6 above)
Comment (sftim) [reply content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md:73]: > When a PR updates an existing page, you should focus on reviewing the parts of the page that are being updated. That changed content should be reviewed for technical and editorial correctness. If you find errors on the page that don't directly relate to what the PR author is attempting to address, then it should be treated as a separate issue (check that there isn't an existing issue about this first).

See https://kubernetes.io/docs/contribute/review/reviewing-prs/#other

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 7]
```diff
@@ -860,9 +860,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `DryRun`: Enable server-side [dry run](/docs/reference/using-api/api-concepts/#dry-run) requests
   so that validation, merging, and mutation can be tested without committing.
 - `DynamicAuditing`: Used to enable dynamic auditing before v1.19.
-- `DynamicKubeletConfig`: Enable the dynamic configuration of kubelet. The
-  feature is no longer supported outside of supported skew policy. The feature
-  gate was removed from kubelet in 1.24. See [Reconfigure kubelet](/docs/tasks/administer-cluster/reconfigure-kubelet/).
+- `DynamicKubeletConfig`: Enable the dynamic configuration of kubelet. (not available for kubelets from Kubernetes v1.24 or later).
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:863]: (nit)
```suggestion
- `DynamicKubeletConfig`: Enable the dynamic configuration of kubelet (not available for kubelets from Kubernetes v1.24 or later).
```

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm

File: content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md [hunk 8]
```diff
@@ -67,7 +67,7 @@ been deprecated. These removals have been superseded by newer, stable/generally
 
 ## API removals, deprecations, and other changes for Kubernetes 1.24
 
-* [Dynamic kubelet configuration](https://github.com/kubernetes/enhancements/issues/281): `DynamicKubeletConfig` is used to enable the dynamic configuration of the kubelet. The `DynamicKubeletConfig` flag was deprecated in Kubernetes 1.22. In v1.24, this feature gate will be removed from the kubelet. See [Reconfigure kubelet](/docs/tasks/administer-cluster/reconfigure-kubelet/). Refer to the ["Dynamic kubelet config is removed" KEP](https://github.com/kubernetes/enhancements/issues/281) for more information. 
+* [Dynamic kubelet configuration:](https://github.com/kubernetes/enhancements/issues/281) `DynamicKubeletConfig` is used to enable the dynamic configuration of the kubelet. The `DynamicKubeletConfig` flag was deprecated in Kubernetes 1.22. In v1.24, this feature gate will be removed from the kubelet. See Reconfigure kubelet. Refer to the ["Dynamic kubelet config is removed" KEP](https://github.com/kubernetes/enhancements/issues/281) for more information.
```
Comment (sftim) [inline content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md:70]: (nit)
```suggestion
* [Dynamic kubelet configuration](https://github.com/kubernetes/enhancements/issues/281): `DynamicKubeletConfig` is used to enable the dynamic configuration of the kubelet. The `DynamicKubeletConfig` flag was deprecated in Kubernetes 1.22. In v1.24, this feature gate will be removed from the kubelet. See Reconfigure kubelet. Refer to the ["Dynamic kubelet config is removed" KEP](https://github.com/kubernetes/enhancements/issues/281) for more information.
```

File: content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md (hunk 5 above)
Comment (Kart121) [reply content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md:72]: @sftim  ohh sorry about that ... so I should create an issue for fixing those right?

Comment (sftim) [reply content/en/blog/_posts/2022-04-07-Kubernetes-1-24-removals-and-deprecations.md:72]: You could log an issue, or open a PR directly. Either's good.

**Comment by sftim** [comment]
@hamees-sayed do you know how to fix the merge conflict? If not, a maintainer can help (with your agreement, we can fix the PR for you).

**Comment by hamees-sayed** [comment]
@sftim Yes, I'd like a maintainer to help me

**Comment by sftim** [comment]
@hamees-sayed we'll try to find you that help

(sorry it took a while)

**Comment by tengqm** [comment]
@Sajiyah-Salat This is not supposed to be merged into `dev-1.26` branch, which is already gone.
Another reason is that the PR was lacking response from the author, for over a month.

A closed PR can be quickly reopened if the author is still interested in working on it.

**Pull request closed without merging**